### DISCUSSION TITLE
I forgot my password

### DISCUSSION THREAD

**Discussion by urmoit** [ente-io/ente#3807] [Q&A]
Hello I forgot my password to enteauth app
And I can't find my recovery key from my computer can someone help me

**Comment by crKRoEYS** [comment]
"What happens if I forget my password?

You can reset your password with your recovery key.

If you lose both your password and your recovery key, you will not be able to decrypt your data."

https://ente.io/faq/security-and-privacy/forgot-password/

But you should mail to: support@ente.io

See https://github.com/ente-io/ente/discussions/3111
